components/autosuggest.md

```diff
@@ -312,6 +367,8 @@ Required: No
 
 Type: String
 
+Default: `'finished'`
```

Comment:
The default value is enclosed in single quotes but the valid values aren't. Is that expected?